### DISCUSSION TITLE
Task 1 lab 3.02

### DISCUSSION THREAD

**Discussion by mateoys** [rramosp/ai4eng.v1#166] [03 - GENERACION DE MODELOS]
Buenas tardes. espero que se encuentren muy bien. Con todo el respeto pido ayuda con el ejercicio 1 del lab 3.02 , lo que pasa es que ejecuto el código y da la salida esperada, sin embargo cuando la plataforma me califica me saca error, quisiera saber cómo lo puedo solucionar , ya que no entiendo que está pasando 
![image](https://user-images.githubusercontent.com/70710614/179069750-8c5ef878-d196-458a-9fd4-6a3782bd6bce.png) este es mi código 
![image](https://user-images.githubusercontent.com/70710614/179069994-6260ad9e-c4fb-47a6-80af-305190052228.png) aquí se puede ver que el código corre.
![image](https://user-images.githubusercontent.com/70710614/179070157-e95a9992-ea8d-46a3-b07d-7c944f3a5660.png)  (aquí el error que me arroja la plataforma)
Por su atención muchas gracias...

**Comment by Felipe-RA** [comment]
Saludos mateoys,

Primero recordarte los lineamientos para hacer preguntas, #145 

No puedo ver tu error completo ni probar tu código, 

Creo que tu problema está en que no permites la creación de múltiples columnas, las quemas directamente en el código.

Recuerda que te dan el `n_timesteps_lookback` para que definas el número columnas `signal` que creas

Hasta pronto!
